cmds/cli/main.go

```diff
@@ -20,14 +20,15 @@ import (
 )
 
 var (
-	v          = flag.Bool("verbose", false, "Verbose output")
-	verbose    = func(string, ...interface{}) {}
-	dir        = flag.String("dir", "", "Path of cached directory")
-	dryRun     = flag.Bool("dryrun", false, "If dry_run is true we won't boot the iso.")
-	distroName = flag.String("distroName", "", "This is the distro that will be tested.")
-	cacheDev   CacheDevice
-	logBuffer  bytes.Buffer
-	tmpBuffer  bytes.Buffer
+	v             = flag.Bool("verbose", false, "Verbose output")
+	verbose       = func(string, ...interface{}) {}
+	dir           = flag.String("dir", "", "Path of cached directory")
+	dryRun        = flag.Bool("dryrun", false, "If dry_run is true we won't boot the iso.")
+	distroName    = flag.String("distroName", "", "This is the distro that will be tested.")
+	webbootDistro = os.GetEnv("WEBBOOT_DISTRO")
```

Comment:
This line is not needed. The distro name is passed in the `distroName` flag instead.